rom/dev/src/rom.ld

```diff
@@ -29,14 +29,14 @@ CFI_STATE_ORG      = 0x500003E4;
 /* Workaround https://github.com/llvm/llvm-project/issues/62423
  * by allowing lld to exceed the ROM size before relaxation optimizations are
  * applied. To prevent legitimate overflows, caliptra_builder::elf2rom() will
- * ensure that the sections fit within the true 48k ROM size.
+ * ensure that the sections fit within the true 96k ROM size.
  */
 ROM_RELAXATION_PADDING = 8k;
 ROM_SIZE          = 96K;
 ICCM_SIZE         = 128K;
 DCCM_SIZE         = 256K;
 DATA_SIZE         = 996;
-STACK_SIZE        = 40K;
+STACK_SIZE        = 46K;
```

Comment:
Why does the stack and ROM size need to change in this PR?

Reply:
So it looks like with the newer toolchain significantly more stack is used. The stack overflow detection code in the emulator tripped on it and this was the minimal increase necessary.